.github/workflows/ci.yml

```diff
@@ -311,6 +311,11 @@ jobs:
           cache: false
       - name: Run build script
         run: make build
+      - name: Upload artifact
```

Comment:
The ci.yml file was refactored recently so keep an eye out for unnecessary steps like this when you rebase.